### PR TITLE
OpenTelemetry in Practice Blog Announcement

### PR DESCRIPTION
**Pull request by musingvirtual**
(With edits by @chalin)

Blog about the OpenTelemetry in Practice event on Aug 23rd

To-do before publishing: 

- [x] add the Zoom link from the OpenTelemetry public calendar - this is stuck behind finding someone who can edit the calendar
  - https://github.com/open-telemetry/community/issues/1130
- [x] Remove the "draft" tag (not doing this until the link is there)
- [x] Resolve TODO items in the page front matter and then remove those comments
- [x] Update the blog publication `date` to the day it's published
- [ ] link this from the end user resources post that is currently in the banner

---

Preview: https://deploy-preview-1618--opentelemetry.netlify.app/blog/2022/otel-in-practice

### CODE REVIEW

**Review by chalin — CHANGES_REQUESTED** [review]
Just a quick comment to drop the `.gitpd.yml` file. I've reopened https://github.com/open-telemetry/opentelemetry.io/issues/1026#issuecomment-1212480071 so that we can track this separately.

**Comment by musingvirtual** [comment]
I think I just removed the yaml file but lmk if that's not the case - I'm pretty new to GitPod. Appreciate you taking time to draw attention to this issue for those of us who don't necessarily want to build the project locally just to change or draft text.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (cartermp) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:14]: Is there a canonical link we can put here for `OpenTelemetry in Practice` for folks?

Comment (cartermp) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:14]: Maybe somewhere in the CNCF slack or something

**Review by cartermp — COMMENTED** [review]
I think this looks good overall, but I'd like to get some links in there. There are also some TODOs to resolve at the top of the file.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (musingvirtual) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:14]: Currently, this blog post is the only thing available about the event and the event is intended to be the first event in a series. I followed the same process Sharr did with the end-user lean coffee - create [blog post](https://opentelemetry.io/blog/2022/new-end-user-resources/) to announce event, get event added to calendar.

What would you suggest that I use as a canonical link? Is there some other material I should create? Let me know and I'll open a PR.

Comment (cartermp) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:14]: Gotcha, makes sense. No real suggestions on my end, so it sounds good to me!

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:49]: I'm not so sure about this sentence:

> Ah, the Collector: it is our friend, but sometimes it is a difficult friend.

I'd suggest dropping it. @cartermp @svrnm - thoughts?

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:55]:
```suggestion
troubleshooting and warn about some pitfalls. Then, share your victories and
frustrations about the Collector, as we learn to be better friends with it,
together.
```

And I suggestion dropping the comment about "friends". 
The "share your victories and frustrations" is good.

**Review by chalin — COMMENTED** [review]
See inline comment, thanks.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (cartermp) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:49]: This is a part of an abstract here: https://www.meetup.com/opentelemetry-in-practice-meetup-group/events/287789310/

So we should probably leave it as-is.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (svrnm) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:5]: feel free to add a link to your github account

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (svrnm) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:28]: Add a link to the channel (https://cloud-native.slack.com/archives/C01RT3MSWGZ) and CNCF slack (https://slack.cncf.io/)

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (svrnm) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:49]: I am with @chalin here, I don't like it, there's lots of room to misinterpret that statement

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (svrnm) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:55]: +1

**Review by svrnm — CHANGES_REQUESTED** [review]

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:49]: I think that it can make sense to use such informality in a meetup announcement. I'd suggest avoiding it here though.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:30]: Change the temporary zoom link to `/`. That will fix the build so that we can see the deploy preview.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (mjingle) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:28]:
```suggestion
reach out in the [#otel-user-research channel](https://cloud-native.slack.com/archives/C01RT3MSWGZ) in the [CNCF Slack](https://slack.cncf.io/).
```

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (mjingle) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:5]: @svrnm How would one add a link to their GitHub account within this frontmatter template? Link or resource is appreciated!

**Comment by musingvirtual** [comment]
PS: scheduled to receive the zoom link in a little so we should be able to publish this tomorrow or later tonight!

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:5]: Use markdown, like this:

```yaml
author: '[Ruben Vargas](https://github.com/rubenvp8510)'
```

which will show up like this: https://opentelemetry.io/blog/2022/k8s-metadata/

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:12]:
```suggestion
## About the series
```

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:14]:
```suggestion
Welcome to the **OpenTelemetry in Practice** series! This is a new experiment by
```

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:22]: Capitalize the first letter of each sentence

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:28]:
```suggestion
future events. So if you’re interested in shaping these conversations,
reach out in the
```

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:30]:
```suggestion
of the [CNCF Slack](https://slack.cncf.io/).
```

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:33]:
```suggestion
the topic is Kubernetes & The Collector.

## Talks

Our lightning talks include the following:
```

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:35]: IMHO, it would be nice to link to presenter pages and/or profiles (e.g. GH), something like this (add a real link):

```suggestion
### Observability Pipelines for Kubernetes

&mdash; by [Daniel Kim](#), New Relic
```

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [inline content/en/blog/2022/OpenTelemetry-In-Practice-August.md:49]: Add a real link below:

```suggestion
### Collector Configuration and Troubleshooting in Kubernetes

&mdash; by [Jessica Kerr](#), Honeycomb
```

**Review by chalin — CHANGES_REQUESTED** [review]
Gave this a more careful review since we're getting closer to publishing. See inline comments, thanks.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (mjingle) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md:22]: Isn't each bullet point a continuation of the sentence that starts with "We're aiming to"? For example, "We're aiming to address..", "We're aiming to build...", etc.?

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md]: Oops, I forgot the quotes here. I'll add that now.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md]: I've shortened the name further.

File: content/en/blog/2022/OpenTelemetry-In-Practice-August.md
Comment (chalin) [reply content/en/blog/2022/OpenTelemetry-In-Practice-August.md]: See, https://developers.google.com/style/capitalization

**Review by chalin — APPROVED** [review]
I've made a final pass and committed a few changes. It now LGTM.

Note the new preview link: https://deploy-preview-1618--opentelemetry.netlify.app/blog/2022/otel-in-practice/

**Comment by chalin** [comment]
All: is this good to go? @svrnm in particular since you had previously asked for some changes.

**Review by cartermp — APPROVED** [review]

**Comment by musingvirtual** [comment]
Believe this is ready for merge once @svrnm approves changes.

The link from the end user resources post I mentioned at the top isn't blocking, I actually need to do that after I land this because it will 404 if I do it before.

thanks everyone!

**Comment by chalin** [comment]
It's the middle of the night for @svrnm. I've double checked that all of his original concerns have been addressed. Would you like for me to merge now @musingvirtual?

**Comment by musingvirtual** [comment]
@chalin thank you so much for all your help with this pr. Yes I think it's
ready to go if you agree!

**Comment by chalin** [comment]
We've got two approvals, and @svrnm's comments have been addresses, so I'll merge now.

**Comment by chalin** [comment]
Thanks all!

**Comment by chalin** [comment]
And it's live: https://opentelemetry.io/blog/2022/otel-in-practice/

**Pull request merged**
